frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -168,6 +168,46 @@ function VirtualKey:init()
             face = self.face,
             bold = self.bold or false,
         }
+    -- make long labeled keys to have a smaller font (c.f. https://github.com/koreader/koreader/pull/7972#issuecomment-884822411)
```

Comment:
Comment indentation is wrong.
`-- Make long labels fit by decreasing font size` (remove the link, not needed, no more info there :)

Reply:
OK!